sensor/common/scan/scan_test.go

```diff
@@ -354,6 +354,7 @@ func (suite *scanTestSuite) TestEnrichThrottle() {
 	scan := LocalScan{
 		scannerClientSingleton: emptyScannerClientSingleton,
 		scanSemaphore:          semaphore.NewWeighted(0),
+		maxSemaphoreWaitTime:   defaultMaxSemaphoreWaitTime,
```

Comment:
Realized this now makes the test take the full timeout team, lets change this to something short to make the test finish faster, such as `1 * time.Millisecond`